.github/workflows/generate-baseline-profile.yml

```diff
@@ -1,16 +1,23 @@
 name: Generate Baseline Profile
 
 on:
+  # Every Sunday at 00:43
+  #  schedule:
+  #    - cron: '43 0 * * 0'
   workflow_dispatch:
+  pull_request:
+    paths:
+      - 'macrobenchmark/**'
+      - '.github/workflows/generate-baseline-profile.yml'
```

Comment:
Ah nice, I didn't know that this way (in case of changes in this file) we can run this Action 👍

Reply:
yep, saw this one also on tivi: https://github.com/chrisbanes/tivi/blob/84383193e0dd79a72bc3a1d9e936869f12f5c443/.github/workflows/baseline-profile.yml#L11